package.json

```diff
@@ -10,10 +10,17 @@
     "postinstall": "nuxt prepare"
   },
   "dependencies": {
+    "@firebase/app-types": "^0.9.1",
     "@mdi/font": "^7.4.47",
+    "firebase": "^10.10.0",
+    "firebase-admin": "^11.11.1",
+    "firebase-functions": "^4.8.2",
     "nuxt": "^3.11.1",
+    "nuxt-vuefire": "^1.0.2",
```

Comment:
If vuefire is no longer needed, can we npm uninstall it? Don't want to make vercel (and other people who fork/use the repo) to install something that isn't actually a dependency.